triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -863,7 +865,7 @@ private[lf] class Runner private (
         .getTransactions(offset, None, filter)
         .map { transaction =>
           triggerContext.childSpan("update") { implicit triggerContext: TriggerLogContext =>
-            triggerContext.logDebug("Transaction source")
+            triggerContext.logDebug("Transaction source", "message" -> transaction)
```

Comment:
Logging a view of the received transaction message allows a ledger view of the ACS to be constructed should rule evaluation stall